### PR TITLE
Improve shutdown server logic

### LINKED ISSUE #170: Batch delete entry during shutting down dmsg server
**Describe the bug**
We need tons of unbind/delete entry during shutting down a dmsg server, because of one by one delete behavior. Could be good if we change that to batch mode.

### PR DESCRIPTION
**Pull request by mrpalide**
Fixes #170 

 Changes:	
- Remove update entry on delete session from server side

How to test this PR:
- Run a dmsg-server based on this PR branch on online instance with `availableSession=3000` in config.
- Clone skywire, checkout to develop, add `time.Sleep(15*time.Minute)` at the beginning of `initLauncher` method at `pkg>visor>init.go` file. Build binaries.
- Copy `underload.sh` from integration folder to skywire cloned folder.
- Remove `sed ...` command line (line 23) in `underload.sh.
- Run tons of visor by `bash underload.sh -n 200 -u dmsgd.skywire.dev` there.
- After all visor connect to dmsg-server, shutdown it with `Ctrl+C`. 
- Should shutdown immediately. 

Note:
I remove update entry of dmsg server when deleting a session. In server shutdown no problem occurs because no need update data in fact. And in disconnect a client (visor) from server, this dmsg server entry update done in internal update at max in next 10 seconds. So no problem occurs here too. Update entry remains same as before for new sessions.

### CODE REVIEW

File: integration/README.md [hunk 1]
```diff
@@ -22,3 +22,30 @@
     ```bash
     $ ./bin/dmsg-server ./integration/configs/dmsgserver1.json
     ```
+
+## Take dmsg-server under load
```
Comment (ersonp) [inline integration/README.md]: `Put dmsg-server under load`

File: integration/README.md [hunk 2]
```diff
@@ -22,3 +22,30 @@
...
+    ```
+    ...
+    func initLauncher(ctx context.Context, v *Visor, log *logging.Logger) error {
+	  time.Sleep(15 * time.Minute)
+	  conf := v.conf.Launcher
+    ...
+    ```
+4. Build new binaries: `make build`
+5. Copy `underLoadLocal.sh` file to skywire clone directory
+6. Run it by `sh underLoadLocal.sh -n 200`
```
Comment (ersonp) [inline integration/README.md]:
```
erson@erson-69:~/Workspace/Go/skywire$ sh underLoadLocal.sh -n 200
underLoadLocal.sh: 3: Syntax error: "(" unexpected
```
this gives error
`bash underLoadLocal.sh -n 200` works

Comment (mrpalide) [reply integration/README.md]: Mh. It work for me with `sh`. I will change it to bash in README.md.

Comment (ersonp) [reply integration/README.md]: the script also is bash `#!/usr/bin/env bash`

**Comment by mrpalide** [comment]
Screen record of test these changes:

https://user-images.githubusercontent.com/79150699/169466678-f2a4f190-d5d9-4342-b12a-7e9e7604412b.mp4

**Review by ersonp — APPROVED** [review]

**Review by jdknives — APPROVED** [review]

**Pull request merged**